lib/data/all_members.dart

```diff
@@ -192,7 +192,7 @@ class AllMembers {
         department: 'Computer Science',
         profilePath: 'assets/profiles/swapnanilray.jpg',
         aboutMe:
-            'Currently in my 2nd year of Engineering, I am highly inclined towards Web Dev. Recently developed a liking for CySec thus working my way out Learning Stuff!',
+            'I am an enthusiastic learner who loves coding and am looking forward to starting to contribute to Open source projects',
```

Comment:
Why did you edit someone else's profile?

Reply:
@imKashyap I did it by mistake , I am new to flutter and open source ,I am sorry.